docs/cpp/source/check-doxygen.sh

```diff
@@ -44,9 +46,12 @@ ignore_warning "warning: explicit link request to 'Item' could not be resolved"
 # Count the number of remaining warnings.
 warnings="$(grep 'warning:' doxygen-log.txt | wc -l)"
 
+echo "Treating all remaining warnings as errors"
+
 if [[ "$warnings" -ne "0" ]]; then
-  echo "Filtered output"
+  echo "Failing Doxygen test because the following warnings were treated fatally:"
   cat doxygen-log.txt
```

Comment:
would it be better output the result of grep 'warning:' instead of the whole log? or is the context useful?

Reply:
I don't really care one way or another, but grepping for warning can lead to some failure modes if done imprecisely (e.g., multiline error message) so I tend to just want to report the log verbatim.,